docs/deploy/ansible/k3s.md

```diff
@@ -0,0 +1,52 @@
+# K3S
+
+*Playbook Path: [ansible/playbooks/install_k3s.yml](https://github.com/webrecorder/browsertrix-cloud/blob/main/ansible/playbooks/install_k3s.yml)*
+
+This playbook provides an easy way to install Browsertrix Cloud on a Linux box (tested on Rocky Linux 9) 
+It automatically sets up Browsertrix with, Letsencrypt certificates.
+
+### Requirements
+
+To run this ansible playbook, you need to:
+
+* Have a server / VPS where browsertrix will run.
```

Comment:
Changed these to `-` instead of `*` in the other files, should probably do the same here.

...Maybe I should look into making a pre-commit hook for docs 👀